extensions/amazon-services/common/deployment/src/main/java/io/quarkus/amazon/common/deployment/AmazonServicesClientsProcessor.java

```diff
@@ -37,6 +38,16 @@ void globalInterceptors(BuildProducer<AmazonClientInterceptorsPathBuildItem> pro
                 new AmazonClientInterceptorsPathBuildItem("software/amazon/awssdk/global/handlers/execution.interceptors"));
     }
 
+    @BuildStep
+    void runtimeInitialize(BuildProducer<RuntimeInitializedClassBuildItem> producer) {
+        producer.produce(
+                new RuntimeInitializedClassBuildItem("software.amazon.awssdk.core.retry.backoff.FullJitterBackoffStrategy"));
+        // This class triggers initialization of FullJitterBackoffStragegy so needs to get runtime-initialized
+        // as well
+        producer.produce(
+                new RuntimeInitializedClassBuildItem("software.amazon.awssdk.services.dynamodb.DynamoDbRetryPolicy"));
```

Comment:
Shouldn't this one be in the `amazon-dynamodb` extension?

Reply:
Probably, I'm having a hard time figuring out where what should go in terms of extensions. I've force-pushed an update. Let me know if that's what you meant or if everything should go into `amazon-dynamodb`...